stable/hazelcast-enterprise/templates/metrics-service.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.metrics.enabled }}
+{{- if and (.Values.hazelcast.enabled .Values.metrics.enabled) }}
```

```suggestion
{{- if and .Values.hazelcast.enabled .Values.metrics.enabled }}
```
PR builder failed due to below lint error:
```
17:03:43 ==> Linting stable/hazelcast-enterprise
17:03:43 [ERROR] templates/: template: hazelcast-enterprise/templates/metrics-service.yaml:1:19: executing "hazelcast-enterprise/templates/metrics-service.yaml" at <.Values.hazelcast.enabled>: enabled is not a method but has arguments
```
I think above change will make green our PR builder.

Also Could you apply same change to https://github.com/hazelcast/charts/blob/master/stable/hazelcast/templates/metrics-service.yaml?